examples/abitest/module_0/rust/src/lib.rs

```diff
@@ -165,30 +170,32 @@ fn status_convert<T>(result: Result<T, OakStatus>) -> std::io::Result<T> {
 }
 
 // Return an offset value that is beyond the bounds of available linear memory.
-fn invalid_raw_offset() -> *mut u64 {
+unsafe fn invalid_raw_offset() -> *mut u64 {
```

Comment:
BTW, why didn't this need to be marked as `unsafe` before?

Reply:
It actually doesn't need to be marked unsafe, would you prefer this reverted? Only dereferencing it is unsafe, but I thought this made the intention more clear.

Reply:
Fine as-is, I just wondered if something had changed to make it necessary that I couldn't see.